examples/hhl.py

```diff
@@ -148,7 +148,7 @@ def _decompose_(self, qubits):
 
 
 class EigenRotation(cirq.Gate):
-    """Perform the of the ancilla equivalent to divison of the memory by eigenvalues of matrix.
+    """Perform the rotation of the ancilla equivalent to divison of the memory by eigenvalues of matrix.
```

Comment:
Lint is yelling about over 100 chars (and pythonistas tend to want single line first sentences).  Maybe

"""Perform a rotation on an ancilla equivalent to division by eigenvalues of a matrix."""